infra/cluster-init/config-features.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-features
+  namespace: knative-serving
+  labels:
+    app.kubernetes.io/name: knative-serving
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/version: devel
+  annotations:
+    knative.dev/example-checksum: "4d5feafc"
+data:
+  kubernetes.podspec-volumes-emptydir: "enabled"
+  kubernetes.podspec-init-containers: "enabled"
```

Comment:
These are the two flags that need to be set to enable those two resources.

Reply:
Super nit: can we name this `knative-configmap.yaml` instead, so it's clear from the filename what it's for?

Reply:
Hmm, or should I just create a folder named `knative-configmaps` and then place this file (still named `config-features.yaml`) inside? Apparently there are like over 10 different Knative configmaps (like `config-autoscaler`, `config-defaults`, `config-logging`, `config-network`, etc.) and it `knative-configmap` might sound a little too generic 👀

Reply:
That works too!